Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -361,7 +373,30 @@ extension TalkPageViewController: TalkPageReplyComposeDelegate {
     }
     
     func tappedPublish(text: String, commentViewModel: TalkPageCellCommentViewModel) {
-        // TODO: Publish reply once live data is connected to commentViewModels
+        viewModel.postReply(commentId: commentViewModel.commentId, comment: text) { [weak self] result in
+
+            switch result {
+            case .success:
+                // TODO: Display success banner
```

Comment:
Is this comment still needed?

Reply:
Nope, good catch! This is removed. I notice there's another one from new topic as well. I'll fix it in my followup PR.